apps/auth-frontend/src/login/Login.tsx

```diff
@@ -7,6 +7,16 @@ import {
 } from '../auth0/web-auth';
 import { extractErrorMessage, WebAuthError } from '../auth0/errors';
 
+const getAppUrl = () => {
+  const redirectUri = new URL(
+    new URLSearchParams(window.location.search).get('redirect_uri') ||
```

Comment:
I think we should throw if `redirect_uri` is not there, to avoid having to hardcode any URL here.
While it is optional in OAuth2, doesn't mean that our flow needs to make it optional, and we already know without a `redirect_uri` the flow will fail because our `add-user-metadata` Auth0 hook throws if it's missing.